Assets/MixedRealityToolkit.Examples/Experimental/Dialog/DialogExampleController.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.Experimental.Dialog;
+
+namespace Microsoft.MixedReality.Examples.Experimental.DialogTest
+{
+    public class DialogExampleController : MonoBehaviour
+    {
+
+        public GameObject dialogPrefab;
```

Comment:
There are a few `public` fields throughout these files. We recommend updating these [into serialized private fields with public properties](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#encapsulation) if code access is needed outside these scripts.